<launch>

    <group ns="bebop">
        <node pkg="med_uav_control" type="height_ctl.py" name="height_ctl" output="screen"/>
        <node pkg="med_uav_control" type="attitude_ctl.py" name="attitude_ctl" output="screen"/>
        <node pkg="med_uav_control" type="horizontal_ctl.py" name="horizontal_ctl" output="screen"/>
    </group>

</launch>
